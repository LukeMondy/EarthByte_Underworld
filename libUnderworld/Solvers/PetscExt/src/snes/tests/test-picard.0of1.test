EXEC 		= ./test-picard.app -n 100 -ksp_type richardson -snes_atol 1.0e-10 -ksp_converged_use_initial_residual_norm
NPROCS 		= 1
TEST_TYPE       = tolerance_diff_pl,1.0e-10
