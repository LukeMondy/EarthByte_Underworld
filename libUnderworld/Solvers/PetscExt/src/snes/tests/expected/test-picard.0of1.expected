In picard_test 
Fnorm(R) = 2.105773e-11 
Newton iterations = 9

